### PR TITLE
chore: bump command version 6 (0.18.0) to command version 7 (0.19.0)

### PR DESCRIPTION
**Pull request by jzaralim**
### Description 
update command version for next release

### Reviewer checklist
- [ ] Ensure docs are updated if necessary. (eg. if a user visible feature is being added or changed).
- [ ] Ensure relevant issues are linked (description should include text like "Fixes #<issue number>")

### CODE REVIEW

File: ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/Command.java [hunk 1]
```diff
@@ -36,7 +36,7 @@
 public class Command {
 
   @VisibleForTesting
-  public static final int VERSION = 6;
+  public static final int VERSION = 7;
```
Comment (spena) [inline ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/Command.java:39]: How is this related to 0.18?

Comment (jzaralim) [reply ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/Command.java:39]: This is used to make sure that old servers don't run new commands. The 0.18 branch is on version 6, so we want master to be on a later version.

**Review by stevenpyzhang — APPROVED** [review]

**Comment by stevenpyzhang** [comment]
Can you make PR title be a bit more clear?
chore: bump command version 6 (0.18.0) to command version 7 (0.19.0)
